Code/Framework/AzCore/AzCore/RTTI/RTTI.h

```diff
@@ -1027,31 +1033,38 @@ namespace AZ
     }
 
     // Returns true if the type is contained, otherwise false. Safe to call for type not supporting AZRtti (returns false unless type fully match).
+
+#if defined(AZ_COMPILER_MSVC)
+    // There is a bug with the MSVC compiler when using the 'auto' keyword here. It appears that MSVC is unable to distinguish between a template
+    // template argument with a type variadic pack vs a template plate argument with a non-type auto variadic pack. 
```

Comment:
repeat: typo with "template plate"

```suggestion
    // template argument with a type variadic pack vs a template template argument with a non-type auto variadic pack. 
```